webapp/graphite/render/datalib.py

```diff
@@ -316,6 +316,11 @@ def prefetchRemoteData(requestContext, targets):
   each time evaluateTarget() needs to fetch a path. All the prefetched
   data is stored in the requestContext, to be accessed later by datalib.
   """
+  # only prefetch if there is at least one remote finder
+  # this is to avoid the overhead of tagdb lookups in extractPathExpressions
+  if len([finder for finder in STORE.finders if not getattr(finder, 'local', True) and not getattr(finder, 'disabled')]) < 1:
```

Comment:
You could do something like this instead of having to build the list of finders and then counting it:
`any(not getattr(finder, 'local', True) and not getattr(finder, 'disabled') for finder in STORE.finders)`